packages/zoe/src/types.js

```diff
@@ -349,17 +347,86 @@
  * @property {Issuer} issuer
  * @property {AmountMath} amountMath
  *
- * @typedef {Object} InstallationRecord
- * @property {InstallationHandle} handle - opaque identifier for the installation
- * @property {SourceBundle} bundle - contract code
- *
- * @typedef {Object} OfferStatus
- * @property {OfferHandle[]} active
- * @property {OfferHandle[]} inactive
- *
- * @typedef {(offerHandle: OfferHandle) => void} OfferHook
+ * @typedef {Record<Keyword,Amount>} Allocation
+ * @typedef {Record<Keyword,AmountMath>} AmountMathKeywordRecord
+ */
+
+/**
+ * @typedef {Object} ZCFSeat
+ * @property {() => void} exit
+ * @property {(msg?: string) => void} kickOut
```

Comment:
The msg is actually a `Details` as defined in assert, rather than a string.

Do we have any candidates for a better name than `kickOut`?